packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -175,8 +175,8 @@ const PersonalDetails = ({
                                                 <FormSubHeader title={localize('Identity verification')} />
                                                 <IDVForm
                                                     selected_country={selected_country}
-                                                    hide_hint={true}
-                                                    can_skip_document_verification={true}
+                                                    hide_hint
+                                                    is_for_new_real_account
```

Comment:
Do you mean "is_for_new_real_account" for real account creation modal?
 can we rename "is_for_new_real_account" for "is_for_real_account_modal"

Reply:
sure Amina, we can rename it 👍